docs/site/index.md

```diff
@@ -1,3 +1,52 @@
 # RecordLinker
 
-The RecordLinker is a service that links records from two datasets based on a set of common attributes. The service is designed to be used in a variety of public health contexts, such as linking patient records from different sources or linking records from different public health surveillance systems. The service uses a probabilistic record linkage algorithm to determine the likelihood that two records refer to the same entity. The service is implemented as a RESTful API that can be accessed over HTTP. The API provides endpoints for uploading datasets, configuring the record linkage process, and retrieving the results of the record linkage process.
+## Control how patient records are matched and merged
+
+<p>
+  <img src="assets/record-linker-diagram.svg" alt="Description" style="float: right; width: 8em; margin-left: 1em; margin-bottom: 1em;">
```

Comment:
I think the alt here needs a better description

Reply:
oops, good call.  I'll get a fix for that in a min

Reply:
fixed.  https://github.com/CDCgov/RecordLinker/pull/374/commits/6f8f116cbcfab2a58ccffb8713f4a8214d93e394